src/main/webapp/populateData.js

```diff
@@ -340,3 +340,105 @@ function populateGo() {
         }
       });
 }
+
+/**
+ * Populate Plan-mail panel with potential times to read the emails
+ */
+function populatePlanMail() {
+  const planContainer = document.querySelector('#plan');
+  fetch('/plan-mail')
+      .then((response) => {
+        // If response is a 403, user is not authenticated
+        if (response.status === 403) {
+          throw new AuthenticationError();
+        }
+        return response.json();
+      })
+      .then((planMailResponse) => {
+        // Display the potential times to create events
+        const numWordsContainer = document.querySelector('#word-count');
+        numWordsContainer.innerText = planMailResponse.wordCount;
+        const averageReplyContainer = document.querySelector('#average-reply');
+        averageReplyContainer.innerText = planMailResponse.averageReadingSpeed;
+        const timeNeededContainer = document.querySelector('#time-needed');
+        timeNeededContainer.innerText = planMailResponse.minutesToRead;
+        const intervalContainer = document.querySelector('#free-interval');
+        intervalContainer.innerHTML = '';
+        for (const index in planMailResponse.potentialMeetingTimes) {
+          if (typeof index == 'string') {
+            const liElement = document.createElement('li');
+            liElement.innerText =
+              planMailResponse.potentialMeetingTimes[index].start +
+              ' ' +
+              planMailResponse.potentialMeetingTimes[index].end;
+            intervalContainer.appendChild(liElement);
+          }
+        }
+      })
+      .catch((e) => {
+        console.log(e);
+        if (e instanceof AuthenticationError) {
+          signOut();
+        }
+      });
+}
+
+/**
+ * Set up the assign panel. For now, this just prints the response
+ * from the server for /gmail-actionable-emails
+ */
+function setUpAssign() {
```

Comment:
I am confused why these assign-related functions are included here. Is this the result of a merge conflict?

Reply:
Yes. I did not implement this. They appeared when updating the branch with master